packages/react-components/react-progress/src/utils/clampMax.test.ts

```diff
@@ -0,0 +1,44 @@
+import { clampMax } from './clampMax';
+
+describe('ProgressBar clampMax util', () => {
+  const originalConsoleError = console.error;
+  beforeEach(() => {
+    console.error = jest.fn();
+  });
```

Comment:
I think you just need to do a `beforeAll` here:

```suggestion
  beforeAll(() => {
    console.error = jest.fn();
  });
```